CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Release History
 
+## 2.7.0 (Unreleased)
+
+### Acknowledgments
+
+Thank you to our developer community members who helped to make the OpenAI client library better with their contributions to this release:
+
+- Benjamin Pinter _([GitHub](https://github.com/BenjaminDavidPinter))_
+
+### Features Added
+
+- OpenAI.Responses:
+  - Enabled support for stable web search functionality, with a separate factory for preview web search features.
```

Comment:
For clarity, mention the `WebSearchTool` class versus the new `WebSearchPreviewTool` class.